tests/e2e/test_redis.py

```diff
@@ -20,107 +20,106 @@ def test_create_redis_collection():
 
 def test_redis_set():
     client = get_client_instance()
```

Comment:
I think we can do client = get_client_instance() only once at the top of the file and then pass client as param to all test functions